frontend/server/src/main/java/org/pytorch/serve/workflow/WorkflowManager.java

```diff
@@ -367,8 +365,14 @@ public WorkFlow getWorkflow(String workflowName) {
 
     public void predict(ChannelHandlerContext ctx, String wfName, RequestInput input)
             throws WorkflowNotFoundException {
+
         WorkFlow wf = workflowMap.get(wfName);
         if (wf != null) {
+            ThreadFactory namedThreadFactory =
+                    new ThreadFactoryBuilder().setNameFormat("wf-manager-thread-%d").build();
+            ExecutorService inferenceExecutorService =
+                    Executors.newFixedThreadPool(
+                            Runtime.getRuntime().availableProcessors(), namedThreadFactory);
```

Comment:
Does this mean a fixed threadpool is generated for each incoming inference request? If yes, it seems a little bit heavy.

Reply:
@lxning Reverted. Although the DagExecuter currently does spin up a thread pool for each of the inference request